tchannel/tornado/tchannel.py

```diff
@@ -212,6 +213,12 @@ def request(self,
 
             Defaults to 'c'.
         """
+        # TODO: do not warn if being called from tchannel.TChannel
```

Comment:
@abhinav I was going to detect if it was being called from `tchannel.TChannel`... is that too nasty?